api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -109,14 +109,13 @@
      *
      * @return a map containing properties of this configuration source
      */
+    @Deprecated
```

Comment:
What would replace this? A @Deprecated annotation without any explanation is always bad. Pretty much prefer the JavaDoc @deprecated

Reply:
This will not be deprecated any more after the further conversation. @radcortez please correct this.

Reply:
The reason why we discussed this to be deprecated is because, potentially computing the getValue for all the properties in the ConfigSource might be too expensive.

This has been discussed before:
https://github.com/eclipse/microprofile-config/issues/431

Reply:
#431 is about the return type of the property name method, not getValue. Could you please elaborate what it has to do with this very line?

Reply:
This was already removed from the PR, so it doesn't make sense to discuss it anymore.